optimization_config.json

```diff
@@ -25,6 +25,12 @@
     "strict": true,
     "comment": "AdGuard German filter"
   },
+  {
+    "filterId": 223,
+    "percent": 50,
+    "strict": true,
+    "comment": "AdGuard Chinese filter"
+  },
   {
     "filterId": 14,
     "percent": 60,
```

Comment:
main annoyances filter must be large. what about decreasing its percent?

Reply:
it's actually not that bad

Reply:
main annoyances filter must be removed:)